docs/build/prove-api/future-updates.md

```
@@ -0,0 +1,52 @@
+
+### Your feedback is valuable to prioritize these features
+
+<br/>
+
+**Passive Event Indexing (Proof cost << 100k)** 
+
+Passive Event/Receipt Indexing compresses the proof size to a single IAVL proof, reducing the cost of verification well below 100k and significantly reducing L1 gas fee. This is achieved by pre-verifying them on the Polymer hub with MMPT and then committing it to our state, so that it is directly accessible via the App-hash.
+
+<br/>
+
+**Proving storage**
```

Markdownlint (0.37.0)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)